preprocessing/nextclade/tests/test_nextclade_preprocessing.py

```diff
@@ -383,6 +387,41 @@ def invalid_sequence() -> str:
             aminoAcidInsertions={},
         ),
     ),
+    Case(
```

Comment:
IMO we don't need this test as it is the same as `name="with one succeeded and one not uploaded"` except that with ebola-sudanuploaded instead of ebola-zaire